Sources/CardPayments/CardClient.swift

```diff
@@ -46,10 +47,14 @@ public class CardClient: NSObject {
             do {
                 let confirmPaymentRequest = try ConfirmPaymentSourceRequest(accessToken: config.accessToken, cardRequest: request)
                 let (result) = try await apiClient.fetch(request: confirmPaymentRequest)
+                
                 if let url: String = result.links?.first(where: { $0.rel == "payer-action" })?.href {
-                    delegate?.cardThreeDSecureWillLaunch(self)
```

Comment:
this just displays message on the demo app, right? Is there reason for deleting this?

Reply:
This wasn't deleted, but moved to line 82. It was moved to line 82, since it is more accurately timed to call this delegate method right before we call `asweb.start()`.

And this is not just for the demo app, this delegate method call to `cardThreeDSecureWillLaunch` will notify a merchant's app that the ASWeb session is about to appear, so they can launch a spinner or handle UI accordingly.

Reply:
We use our Demo app as a sample merchant app, essentially.